src/app/js/formats/asterPlotChart/AsterPlotChartView.spec.js

```diff
@@ -0,0 +1,45 @@
+import { getPercentValue } from './AsterPlotChartView';
+
+describe('AsterPlotChartView', () => {
+    describe('getValue', () => {
+        it('should return 0 if weight is wrong', () => {
+            const test = data => {
+                const result = getPercentValue(data);
+                expect(result).toBe('0');
+            };
+
+            const wrongData = [
```

Comment:
In unit testing, you should test only one wrong data, so split it is several tests